docs/modules/servers/pages/postgres/run/run-docker.adoc

```diff
@@ -0,0 +1,145 @@
+= Postgresql James Server &mdash; Run with docker
+:navtitle: Run with docker
+
+== Running via docker-compose
+
+Requirements: docker & docker-compose installed.
+
+When you try James this way, you will use the most current state of James.
+
+=== Running with Postgresql only
+
+It will be configured to run with Postgresql.
+All those components will be started with a single command.
+
+You can retrieve the docker-compose file : ( docker-compose file and james image name should be changed)
+
+    $ wget https://raw.githubusercontent.com/apache/james-project/master/server/apps/postgres-app/docker-compose.yml
```

Comment:
Should we be concerned about the "/master/" branch here?
(postgres-app does not exits in master branch currently)

Reply:
Good point. But it will be the case when we merge it to master though, and I'm afraid we forget it then